x-pack/plugins/cases/public/components/user_profiles/user_tooltip.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiFlexGroup, EuiFlexItem, EuiText, EuiToolTip } from '@elastic/eui';
+import { UserProfileUserInfo, UserProfileWithAvatar } from '@kbn/user-profile-components';
+import { CaseUserAvatar } from './user_avatar';
+import { getName } from './display_name';
+import * as i18n from './translations';
+
+const UserFullInformation: React.FC<{ profile?: UserProfileWithAvatar }> = React.memo(
+  ({ profile }) => {
+    if (profile?.user.display_name && profile?.user.full_name) {
+      return (
+        <>
+          <EuiText
+            size="s"
+            className="eui-textBreakWord"
+            data-test-subj="user-profile-tooltip-display-name"
+          >
+            <strong>{profile.user.display_name}</strong>
+          </EuiText>
+          <EuiText
+            size="s"
+            className="eui-textBreakWord"
+            color="subdued"
+            data-test-subj="user-profile-tooltip-full-name"
+          >{`(${profile.user.full_name})`}</EuiText>
+        </>
```

Comment:
Would it make sense to combine this in a single `EuiText` component? Something like this:

```suggestion
        <EuiText
          size="s"
          className="eui-textBreakWord"
        >
          <strong data-test-subj="user-profile-tooltip-display-name">{profile.user.display_name}</strong>{' '}
          <EuiTextColor 
            color="subdued"
            data-test-subj="user-profile-tooltip-full-name"
          >
            {`(${profile.user.full_name})`}
          </EuiTextColor>
        </EuiText>
```